sympy/core/tests/test_match.py

```diff
@@ -472,10 +472,10 @@ def test__combine_inverse():
     assert Mul._combine_inverse(oo*I*y, oo*I) == y
     assert Mul._combine_inverse(oo*y, -oo) == -y
     assert Mul._combine_inverse(-oo*y, oo) == -y
-    assert Add._combine_inverse(oo, oo) == S(0)
-    assert Add._combine_inverse(oo*I, oo*I) == S(0)
-    assert Add._combine_inverse(x*oo, x*oo) == S(0)
-    assert Add._combine_inverse(-x*oo, -x*oo) == S(0)
+    assert Add._combine_inverse(oo, oo) == S.Zero
```

Comment:
` == S.Zero\n` -> ` is S.Zero\n` globally

Reply:
Good way to find bugs. Turned out the the stats module used `sum` which returns an `int` if everything summed up is 0.